dashboard/15-final/app/ui/table.tsx

```diff
@@ -0,0 +1,62 @@
+import {invoices} from "../lib/dummy-data";
+import Link from "next/link";
+
+export default function Example() {
+  return (
+    <div className="w-full">
+      <div className="w-full flex items-center justify-between">
+        <h1 className="text-base font-semibold leading-6 text-gray-900">Invoices</h1>
+        <Link
+          href="/dashboard/invoices/create"
+          className="block rounded-md bg-indigo-600 px-3 py-2 text-center text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+        >
+          Add Invoice
+        </Link>
+      </div>
+      <div className="mt-8 ">
+        <div className="overflow-x-auto">
+          <div className="overflow-hidden border rounded-md">
+            <table className="min-w-full divide-y divide-gray-300">
+              <thead className="bg-gray-50">
+                <tr>
+                  <th scope="col" className="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-6">
+                    ID
+                  </th>
+                  <th scope="col" className="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
+                    Customer ID
```

Comment:
Let's add a customer name column here, this is when things might get tricky because we need to use the `customerId` from the invoices table to get the `name` from the customer table.